client.go

```diff
@@ -107,14 +106,16 @@ func newRPC2Client(conn net.Conn) (*OvsdbClient, error) {
 
 	// Process Async Notifications
 	dbs, err := ovs.ListDbs()
-	if err == nil {
-		for _, db := range dbs {
-			schema, err := ovs.GetSchema(db)
-			if err == nil {
-				ovs.Schema[db] = *schema
-			} else {
-				return nil, err
-			}
+	if err != nil {
+		return nil, err
+	}
+
+	for _, db := range dbs {
+		schema, err := ovs.GetSchema(db)
+		if err == nil {
+			ovs.Schema[db] = *schema
+		} else {
+			return nil, err
```

Comment:
nit. something I found useful is to aggregate errors here and check after the loop if there was an error and return them so, if there are different errors with the schema, you can spot all of them. Otherwise, you have to rerun the program multiple times until you fix all the errors one by one

Reply:
Done, thanks